packages/engine/src/interaction/systems/MountPointSystem.ts

```diff
@@ -199,10 +203,10 @@ const vec3_1 = new Vector3()
 const reactor = () => {
   const mountedEntities = useHookstate(getMutableState(MountPointState))
   useEffect(() => {
-    //temporary logic for setting visibility of hints until interactive system is refactored
+    //temporary logic for setting visibility of hints until interactable system is refactored
```

Comment:
This comment probably doesn't apply anymore

Reply:
noting that i want to revisit how we handle hiding, as this is still currently required to hide as-written and if feels like a hack workaround

Reply:
no longer relevant with new system design